roles/rook/README.rst

```diff
@@ -0,0 +1,101 @@
+Rook
+====
+
+This role is used for updating and recovering the rook jenkins in the rook ci vpc.
+
+Those are the functions in this role are
+rook-jenkins-update - Updating the jenkins app to the version that is defined in the "jenkins_master_image" varible
+rook-os-update - Updating the packages of the rook jenkins OS
+rook-recovery - Recoverying the Prod-jenkins from the image defined in the "image" varible in a case that the instance was deleted
```

Comment:
Find a way to nicely display the role names and functions so they're not all the same font.  Maybe bullet points with the role names bold.